vividus-studio-server/vividus-studio-plugin/src/main/java/org/vividus/studio/plugin/finder/StepDefinitionFinder.java

```diff
@@ -237,6 +237,7 @@ private StepDefinition createStepDefinition(String module, String stepAsString,
         return new StepDefinition(module, stepAsString, documentation, parameters);
     }
 
+    @SuppressWarnings("PreferMethodReference")
```

Comment:
at some degree related to https://bugs.java.com/bugdatabase/view_bug.do?bug_id=8142476 but still can be reproduced on jdk 13
```
java.lang.BootstrapMethodError: bootstrap method initialization exception
	at java.base/java.lang.invoke.BootstrapMethodInvoker.invoke(BootstrapMethodInvoker.java:194)
	at java.base/java.lang.invoke.CallSite.makeSite(CallSite.java:307)
	at java.base/java.lang.invoke.MethodHandleNatives.linkCallSiteImpl(MethodHandleNatives.java:259)
	at java.base/java.lang.invoke.MethodHandleNatives.linkCallSite(MethodHandleNatives.java:249)
	at org.vividus.studio.plugin.finder.StepDefinitionFinder.children(StepDefinitionFinder.java:242)
	at org.vividus.studio.plugin.finder.StepDefinitionFinder.find(StepDefinitionFinder.java:80)
	at org.vividus.studio.plugin.finder.StepDefinitionFinderTests.testFind(StepDefinitionFinderTests.java:118)
```